### PR TITLE
adding static fields for the NFT event

### PR DESCRIPTION
**Pull request by julien51**
In order to be able to release this ealier than later I hard coded some values so that we render things correctly!

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: tickets/src/components/content/EventContent.js [hunk 1]
```diff
@@ -98,8 +106,14 @@ export class EventContent extends Component {
             <Row>
               <DetailsField>
                 <DisplayDate>{dateString}</DisplayDate>
-                <Description>{description}</Description>
+                <Description>
+                  {description.split('\n').map(line => {
```
Comment (benwerd) [inline tickets/src/components/content/EventContent.js]: Should this be `\n\n`? In some ways I'd be inclined to replace the paragraph tags with `<br />` - right now a single carriage return would mean a new paragraph.

File: tickets/src/components/content/EventContent.js [hunk 2]
```diff
@@ -186,6 +199,25 @@ export const mapStateToProps = (
       description,
       location,
     }
+    // TEMPORARY: HARD CODE VALUES FOR NFT EVENT
+    if (lockAddress === '0x5Cd3FC283c42B4d5083dbA4a6bE5ac58fC0f0267') {
```
Comment (benwerd) [inline tickets/src/components/content/EventContent.js]: Don't you mean `0x5865Ff2CBd045Ef1cfE19739df19E83B32b783b4`?

Comment (julien51) [reply tickets/src/components/content/EventContent.js]: Oups! thanks for the sharp eye!

**Pull request merged**